Java/avg/array.java

```diff
@@ -12,4 +12,15 @@ public static void main(String[] args) {
             System.out.println(arr[i]);
         }
     }
+
+    public static void Even(String[] args) {
+        Scanner sc = new Scanner(System.in);
+        int[] arr = new int[]{1, 2, 3, 4, 5};
+        System.out.println("Even numbers:");
+
+        for (int i = 0; i < arr.length; i = i * 2)
+        {
+            System.out.println(arr[i]);
+        }
+    }
 }

```

Comment:
**Suggestion**: Add unit tests for the "printEvenNumbers" method to ensure its correct functionality and to facilitate future maintenance and refactoring. <br> **Code Suggestion**: 
 ```
 +    // Unit tests should be added to validate the functionality of printEvenNumbers
 ```